### PR TITLE
chore(release): release 9.0.0-beta.1

### PR DESCRIPTION
**Pull request by vthinkxie**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: docs/changelog.zh-CN.md [hunk 1]
```diff
@@ -13,6 +13,121 @@ timeline: true
...
+* **input-number:** 支持 nzPrecisionMode 模式 ([#4185](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4185)) ([bfe089f](https://github.com/NG-ZORRO/ng-zorro-antd/commit/bfe089f)), closes [#4173](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4173)
+* **input-number:** 支持 ngModelChange 在输入时立即触发 ([#4769](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4769)) ([299ba6d](https://github.com/NG-ZORRO/ng-zorro-antd/commit/299ba6d)), closes [#3039](https://github.com/NG-ZORRO/ng-zorro-antd/issues/3039) [#3773](https://github.com/NG-ZORRO/ng-zorro-antd/issues/3773)
+* **menu:** 当 sider 收起时，自动触发 nzInlineCollapsed模式 ([51fbf5e](https://github.com/NG-ZORRO/ng-zorro-antd/commit/51fbf5e)), closes [#4680](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4680)
+* **menu:** 支持 nzMatchRouter 与 CanDeactivate 配合工作 ([7560563](https://github.com/NG-ZORRO/ng-zorro-antd/commit/7560563)), closes [#4407](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4407)
+* **monaco-editor:** 支持静态导入 ([#4341](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4341)) ([29f732b](https://github.com/NG-ZORRO/ng-zorro-antd/commit/29f732b))
```
Comment (wendellhu95) [inline docs/changelog.zh-CN.md]: code-editor

File: docs/changelog.zh-CN.md [hunk 2]
```diff
@@ -13,6 +13,121 @@ timeline: true
...
+* **input:** 提升 input 性能 ([7af643b](https://github.com/NG-ZORRO/ng-zorro-antd/commit/7af643b)), closes [#3950](https://github.com/NG-ZORRO/ng-zorro-antd/issues/3950)
+* **radio:** 重构了数据流 ([#4770](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4770)) ([423a382](https://github.com/NG-ZORRO/ng-zorro-antd/commit/423a382))
+
+
+### BREAKING CHANGES
+
+注意： 所有不兼容改动均在 8.x 最新版本中给出了 warning，如果你在 8.x 最新版本中修复了所有 warning，9.x 可以直接升级的，`ng update` 工具在 9.0 正式版会提供。
+
+* **icon:** 
+  - `NZ_ICON_DEFAULT_TWOTONE_COLOR` 被移除， 请使用 `NzGlobalConfigService`。
```
Comment (wendellhu95) [inline docs/changelog.zh-CN.md]: icon 有两条，建议和下面的合并。

File: docs/changelog.zh-CN.md [hunk 3]
```diff
@@ -13,6 +13,121 @@ timeline: true
 * 主版本号：含有破坏性更新和新特性，不在发布周期内。
 
 ---
+## 9.0.0-beta.1
+
+`2020-03-15`
+
+### Bug Fixes
+
+* **code-editor:** 修复 diff 模式下的问题 ([#4485](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4485)) ([#4532](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4532)) ([021cf22](https://github.com/NG-ZORRO/ng-zorro-antd/commit/021cf22))
```
Comment (wendellhu95) [inline docs/changelog.zh-CN.md]: 建议排序到下面。

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,149 @@
...
+* **checkbox:** use css empty selector instead of observeContent ([#4761](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4761)) ([da8821a](https://github.com/NG-ZORRO/ng-zorro-antd/commit/da8821a))
+* **input:** improve input-group perf ([7af643b](https://github.com/NG-ZORRO/ng-zorro-antd/commit/7af643b)), closes [#3950](https://github.com/NG-ZORRO/ng-zorro-antd/issues/3950)
+* **radio:** refactor radio group data flow ([#4770](https://github.com/NG-ZORRO/ng-zorro-antd/issues/4770)) ([423a382](https://github.com/NG-ZORRO/ng-zorro-antd/commit/423a382))
+
+
+### BREAKING CHANGES
+
+* **icon:** - `NZ_ICON_DEFAULT_TWOTONE_COLOR` is removed. Use `NzGlobalConfigService` instead.
+
+chore: remove strange file
```
Comment (wendellhu95) [inline CHANGELOG.md]: 移除 commit message。

File: docs/changelog.zh-CN.md [hunk 5]
```diff
@@ -13,6 +13,121 @@ timeline: true
...
+* **carousel:** Carousel
+  - `nzVertical` 被移除，请使用 'nzDotPosition' 代替。
+* **icon:** 
+  - `i[nz-icon]`:  `twoToneColor` `theme` `spin` `iconfont` `type` 输入被移除, 请使用 `nzTwoToneColor` `nzTheme` `nzSpin` `nzIconfont` `nzType` 代替。
+  - `i.anticon` 被移除, 请使用 `i[nz-icon]` 代替。
+* **calendar:** 
+  - `<nz-calendar>` `nzCard` 被移除了, 请使用 `nzFullscreen` 代替。
+* **tooltip,popover,popconfirm:** 
+  -`<nz-tooltip>` `<nz-popover>` `<nz-popconfirm>` 组件被移除, 请使用对应的 directives 代替。
+* 移除了无用的 API `NgZorroAntdModule.forRoot()`
```
Comment (wenqi73) [inline docs/changelog.zh-CN.md]: * **date-picker:**
  - date-fns 模式下 `[nzFormat]` 发生[改变](https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/docs/i18n.en-US.md#date-fns-update)。

**Pull request merged**